mobile/lib/network/api_managers/movie_api_manager.dart

```diff
@@ -0,0 +1,25 @@
+ class MovieAPIManager{
+  
+  static MovieAPIManager _instance; 
+  static String _movieKey;
+  static String _movieBaseURL;
+  static String _moviePosterBaseURL;
+
+  factory MovieAPIManager(){
+    _instance ??= MovieAPIManager._internal();
+    _movieKey = 'db09af0e041def3d0404bdefc82962f7';
```

Comment:
WHOAAAA

Reply:
Look into how to use .env of some sort to hide credentials

Reply:
handled in latest commit